core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4770,6 +4779,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String NETWORK_HOST_RESOLUTION_TIMEOUT_MS =
         "alluxio.network.host.resolution.timeout";
     public static final String SITE_CONF_DIR = "alluxio.site.conf.dir";
+    public static final String SERVICE_USE_IP = "alluxio.service.use.ip";
```

Comment:
name suggestion: 
`alluxio.network.ip.address.used`

Reply:
Thanks for the name suggestion!